implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v1/DynatraceNamingConvention.java

```diff
@@ -30,7 +30,7 @@
  * @author Johnny Lim
  * @since 1.1.0
  */
-public class DynatraceNamingConvention implements NamingConvention {
+class DynatraceNamingConvention implements NamingConvention {
```

Comment:
The other naming conventions are public (also, this is a breaking change but I'm not sure if it is really used).
@shakuzen Do you happen to know why are they public and if moving this is fine?

Reply:
They are public because users may want to extend the naming convention that we provide and use by default with some extra logic of their own. We would not want to break this in a minor release.